src/view/drag-handle/util/create-visibility-event.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import type { EventBinding } from './event-types';
+
+const getVisibiltyEvent = () : ?{hidden:string, name:string} => {
+  const prefixes = ['ms', 'webkit', 'moz', 'o'];
+  let browserPrefix;
+  prefixes.forEach((prefix) => {
+    if (`${prefix}Hidden` in document) {
+      browserPrefix = {
+        hidden: `${prefix}Hidden`,
+        name: `${prefix}visibilitychange`,
+      };
+    }
+  });
+  if (!browserPrefix && 'hidden' in document) { // Opera 12.10 and Firefox 18 and later support
+    return {
+      hidden: 'hidden',
+      name: 'visibilitychange',
+    };
+  }
+  return browserPrefix;
+};
+
+export default (cancel: Function = () => {}) : EventBinding => {
+  if (typeof window !== 'undefined') {
```

Comment:
When can this function be called when window is not around?

Reply:
In the integration tests for SSR
Within `test/unit/integration/server-side-rendering.spec.js` the following tests fail
 -  should render identical content when resetting context between renders
 -  should support rendering to static markup
 -  should support rendering to a string